csharp/README.dist

```diff
@@ -3,18 +3,17 @@ XenServer.NET
 
 Version @SDK_VERSION@
 
-XenServer.NET is a complete SDK for Citrix XenServer,
-exposing the XenServer API as .NET classes. It is written in C#.
+XenServer.NET is a complete SDK for the Citrix Hypervisor,

```

Comment:
Should "the" be removed? We are talking about the product "Citrix Hypervisor" itself, not something like " the Citrix Hypervisor API".